### PR TITLE
AB#425 US424-TA425: Split home rendering

### PR DESCRIPTION
**Pull request by yuya-yuliya**
- add RecommendedProducts and BannerGrid renderings
- add renderingWithContextAndDatasource

### CODE REVIEW

File: src/Feature/PageContent/client/Home/index.tsx [hunk 1]
```diff
@@ -14,24 +14,21 @@
...
-                <HeroHome {...this.props} />
-                <SliderHome {...this.props} />
-            </div>
-        );
-    }
+class HomeControl extends JSS.SafePureComponent<HomeControlProps, HomeControlState> {
+  public safeRender() {
+    return (
+      <div className="container">
+        <Placeholder name="banner" rendering={this.props.rendering} />
```
Comment (larionovadaria) [inline src/Feature/PageContent/client/Home/index.tsx:27]: Change to 1 placeholder and place 2 renderings in it

File: src/Feature/PageContent/client/RecommendedProducts/index.tsx [hunk 2]
```diff
@@ -14,16 +14,16 @@
...
 import { SliderHomeControlProps, SliderHomeControlState } from './models';
 
-import * as Jss from 'Foundation/ReactJss/client';
+import * as JSS from 'Foundation/ReactJss/client';
 import './styles.scss';
 
 import GlassesImage from 'Foundation/UI/client/common/media/images/glasses-for-slider.png';
 
-class SliderHomeControl extends Jss.SafePureComponent<SliderHomeControlProps, SliderHomeControlState> {
+class RecommendedProductsControl extends JSS.SafePureComponent<SliderHomeControlProps, SliderHomeControlState> {
```
Comment (larionovadaria) [inline src/Feature/PageContent/client/RecommendedProducts/index.tsx:26]: This is `RecommendedProductsComponent`

File: src/Feature/PageContent/client/BannerGrid/index.tsx [hunk 3]
```diff
@@ -30,7 +29,7 @@ import {
 
 import './styles.scss';
 
-class HeroHomeControl extends Jss.SafePureComponent<HeroHomeControlProps, HeroHomeControlState> {
```
Comment (larionovadaria) [inline src/Feature/PageContent/client/BannerGrid/index.tsx:33]: Banner grid is a rendering with placeholders in it.

**Review by ghost — CHANGES_REQUESTED** [review]

File: src/Feature/PageContent/client/BannerGrid/index.tsx (hunk 3 above)
Comment (yuya-yuliya) [reply src/Feature/PageContent/client/BannerGrid/index.tsx:33]: Yes, this change will added in another task for this story

**Review by ghost — APPROVED** [review]

**Pull request merged**